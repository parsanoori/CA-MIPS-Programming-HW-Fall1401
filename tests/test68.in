49
691293206
550297536
1000197710
797933800
832615612
1008263670
440507418
361311001
112977285
38253140
265711838
87771654
730962067
87310709
756118211
910319858
304204561
70550993
433737555
264645690
1033237279
1059084995
278034743
160655964
34958981
918981559
489196119
898558433
421344813
542027411
654764188
523389606
525911582
594098063
441281247
540878172
748831688
1029006264
613528950
981372601
728639966
259167376
439961409
881220446
653601818
263793755
826164431
580738905
562908135