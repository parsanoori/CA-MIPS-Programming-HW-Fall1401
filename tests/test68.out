34958981
38253140
70550993
87310709
87771654
112977285
160655964
259167376
263793755
264645690
265711838
278034743
304204561
361311001
421344813
433737555
439961409
440507418
441281247
489196119
523389606
525911582
540878172
542027411
550297536
562908135
580738905
594098063
613528950
653601818
654764188
691293206
728639966
730962067
748831688
756118211
797933800
826164431
832615612
881220446
898558433
910319858
918981559
981372601
1000197710
1008263670
1029006264
1033237279
1059084995